### PR TITLE
Native Stimulus 3 Support

### PR DESCRIPTION
**Pull request by adrienpoly**
mostly replace stimulus imports by @hotwired/stimulus

this version can be tested by installing v0.50.0-0 beta version 

```
yarn add stimulus-use@beta
```

### CODE REVIEW

**Review by weaverryan — COMMENTED** [review]
Thanks for getting this going - an annoying detail - we're doing the same over in the symfony packages :)

File: package.json [hunk 1]
```diff
@@ -81,7 +85,8 @@
     "url": "https://github.com/stimulus-use/stimulus-use"
   },
   "peerDependencies": {
-    "stimulus": ">=1.1.1 <3"
+    "@hotwired/stimulus": ">=3",
+    "stimulus": ">=3"
```
Comment (davidjr82) [inline package.json:89]: Why to add peerDependency "stimulus": ">=3"? 

If the user has installed "@hotwired/stimulus" then a warning will raise up:

`warning " > stimulus-use@0.50.0-2" has unmet peer dependency "stimulus@>=3".`

**Comment by fedegl** [comment]
Hello @adrienpoly. Thanks for the awesome library!

As others have commented, I am too waiting on the new release to be able to upgrade to Stimulus 3.0. I understand that it's on you own time and you might be busy. But if you could just give us a bit of an expectation on when are you planning on releasing this update, it would help to make a decision on what to do with regards to this library.

Thanks!

**Comment by abunch** [comment]
Any update on when this will get merged in?

**Comment by janklan** [comment]
I'm not sure where is the best place to ask, so I'll ask here: Is this repo dead?

**Comment by marcoroth** [comment]
Hey @janklan, it's definitely not dead. We are looking to get this merged in asap, there were just a lot of moving parts in the Hotwire space.

**Comment by janklan** [comment]
> Hey @janklan, it's definitely not dead. We are looking to get this merged in asap, there were just a lot of moving parts in the Hotwire space.

Phew, that's a relief. I'm looking at ditching AlpineJS for Stimulus in my Symfony application, and this package seems to be a make-or-brake part in doing so. Last commit 7 months ago is usually a sign of bad things on the horizon.

Is there a timeline? 

I know it's open source, people invest their time etc etc., so I'm not trying to be pushy in any way, I just need to make an executive decision to either dive in the beta, or hold back.

**Comment by marcoroth** [comment]
> Is there a timeline?
There is no official timeline. 

But I'm planning to work on getting the library updated in this coming week, there are a few chores which are overdue.

**Comment by janklan** [comment]
Cool, thanks for the update!

**Comment by weaverryan** [comment]
@marcoroth If you're able to list any of those chores... and the rest of us can help, seriously let us know :). Thanks for your efforts - looking forward to the non-beta 0.50 release!

File: package.json (hunk 1 above)
Comment (marcoroth) [reply package.json]: Since quite some time passed in the meantime I removed the explicit `peerDependencies` on `stimulus`

**Comment by marcoroth** [comment]
Thanks for your offer @weaverryan. I think at this point it doesn't really make sense to split up the work. But I really appreciate it! 

We are super close to release the final version!

**Comment by marcoroth** [comment]
Good news! We just released [`0.50.0`](https://github.com/stimulus-use/stimulus-use/releases/tag/v0.50.0) which ships native support for Stimulus 3, thanks your for patience!

You should also be able to remove the `stimulus` npm package from your `package.json`, given no other package depends on it.

Please let us know if you see anything unexpected, thank you!

**Comment by janklan** [comment]
The beat news to read the first thing in the morning. Thanks!

**Pull request merged**